client/context/context.go

```diff
@@ -16,6 +16,7 @@ import (
 	"github.com/cosmos/cosmos-sdk/x/auth"
 
 	"github.com/spf13/viper"
+	yaml "gopkg.in/yaml.v2"
```

Comment:
I don't think we need this alias, the lib should work with `yaml.Xxx` even if we don't include this alias. (comment applies to all imports like this)

Reply:
Blame VSCode for that :) will fix it, thanks!